football_now.rb

```diff
@@ -11,50 +12,69 @@ def start
-        end
 
-        def datetime_adjust(date)
-            date_splitted = date.split(' ').drop(1)
+        def city_prompt
+            puts "\nPORTUGUESE SPORTV TV CHANNELS"
+            puts "\nEnter the city to convert games date/times into"
+            puts "\nNOTE: type #{Paint['help', :green]} for the list of cities/timezones\n"
 
-            new_time = offset(date_splitted, 'Berlin')
-            date = date_adjust(date_splitted, new_time)
+            user_input
+        end
 
-            "#{date} #{new_time}"
+        def user_input
+            input = gets.chomp
+            input.downcase == 'help' ? cities_tz_list : @city_tz = find_tzinfo(input.capitalize)
         end
 
-        def offset(date_splitted, city_tz)
-            Time.zone = 'London'
-            game_bst_time = Time.zone.parse("#{date_splitted[2]}-#{date_splitted[1]}-#{date_splitted.first} #{date_splitted.last}")
+        def cities_tz_list
+            ActiveSupport::TimeZone.all.sort_by { |o| o.name }.map {|o| puts o.name} 
+            city_prompt
+        end
+
+        def find_tzinfo(input)
+            ActiveSupport::TimeZone.find_tzinfo(input)
 
-            tz = ActiveSupport::TimeZone.find_tzinfo(city_tz)
-            game_cest_time = game_bst_time.in_time_zone(tz)
+        rescue TZInfo::InvalidTimezoneIdentifier => e
+            puts "\n\n#{Paint['We could not find in out timezone list:', :red]} #{e.message}"
+            city_prompt
+        end
+
+        def print_games(games)
+            puts "\nCity timezone of: #{@city_tz.name}: #{@city_tz.now.strftime('%d/%m %H:%M')}\n\n"
 
-            "#{game_cest_time.strftime('%H:%M')}#{Paint[' •', :green, :blink] if is_live?(game_cest_time, tz)}"
+            games.each do |game|
+                puts_game(game)
+            end
         end
 
-        def is_live?(game_cest_time, tz)
-            (game_cest_time..game_cest_time.change(hour: game_cest_time.hour + 2)).cover? Time.now.in_time_zone(tz)
+        def puts_game(game)
+            game_splitted = game['title'].split(' - ')
+        
+            puts %Q(#{ offset(game['pubDate']) } #{Paint[game_splitted.last, :white, :italic]} - #{ Paint[colorize_benfica(game_splitted.first), :bold] }\n)
         end
 
-        def date_adjust(date_splitted, new_time)
-            methods = []
-            new_time == '00:00' ? methods << [:days_since, 1] : nil
-            methods << [:strftime, '%d/%m']
+        def offset(pubDate)
+            date_splitted = pubDate.split(' ').drop(1)
+
+            Time.zone = 'London'
+            game_bst_time = Time.zone.parse("#{date_splitted[2]}-#{date_splitted[1]}-#{date_splitted.first} #{date_splitted.last}")
+
+            game_user_time = game_bst_time.in_time_zone(@city_tz)
+
+            "#{game_user_time.strftime('%d/%m %H:%M')}#{Paint[' •', :green, :blink] if is_live?(game_user_time)}"
```

Comment:
This is a little puts_game to me instead of offset.

Reply:
Didnt want to make a multiner puts and neither a more long one liner by passing the is_live? call and condition to there . Will have a look later.

Reply:
So would have to look if a • is present, if it is to be Paint as green inside the puts game, can be done. Thks